mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -226,9 +226,6 @@ def run(self):  # {{{
                 startDate=startDateFirstYear,
                 endDate=endDateFirstYear)
 
-            dsFirstYear = \
```

Comment:
This part of the code only gets called if `startYear > 1` for time series, so I didn't catch it in my testing.  I do believe the bug was introduced in the effort to remove mapping files and such.  The `ncclimo` work wouldn't have touched this file.